lib/routes/xueqiu/timeline.ts

```diff
@@ -0,0 +1,88 @@
+import { Route } from '@/types';
+import got from '@/utils/got';
+import { config } from '@/config';
+import { parseDate } from '@/utils/parse-date';
+import cache from '@/utils/cache';
+const rootUrl = 'https://xueqiu.com';
+export const route: Route = {
+    path: '/timeline',
+    categories: ['finance'],
+    example: '/xueqiu/timeline',
+    parameters: {},
+    features: {
+        requireConfig: [
+            {
+                name: 'XUEQIU_COOKIES',
+                description: '',
+            },
+        ],
+        requirePuppeteer: false,
+        antiCrawler: true,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    name: '用户关注时间线',
+    maintainers: ['ErnestDong'],
+    handler,
+    description: `:::warning
+  用户关注动态需要登录后的 Cookie 值，所以只能自建，详情见部署页面的配置模块。
+  :::`,
+};
+
+async function handler(ctx) {
+    const cookie = config.xueqiu.cookies;
+    const limit = ctx.req.query('limit') || 15;
+    const usergroup_id = ctx.req.query('usergroup_id') || 1;
+    if (cookie === undefined) {
+        throw new Error('缺少雪球用户登录后的 Cookie 值');
+    }
+    let out = [] as any[];
```

Comment:
Do not use `any`
https://github.com/DIYgod/RSSHub/blob/d748a8c92e714a5bea9526ed9abf8191c487ccab/lib/types.ts#L30